app/models/miq_expression.rb

```diff
@@ -775,58 +775,58 @@ def _to_sql(exp, tz)
     clause
   end
 
-  def preprocess_for_sql(exp, attrs = nil)
+  def preprocess_for_sql(expr, attrs = nil)
```

Comment:
I agree with @chessbyte here - using `expression` over `expr` here is much more expressive :wink: